pom.xml

```diff
@@ -51,26 +51,26 @@
     </developers>
 
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/vsphere-cloud-plugin.git</connection>
-        <developerConnection>scm:git:ssh://git@github.com/jenkinsci/vsphere-cloud-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/vsphere-cloud-plugin</url>
+        <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+        <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+        <url>https://github.com/${gitHubRepo}</url>
         <tag>${scmTag}</tag>
     </scm>
 
     <properties>
         <revision>2.28</revision>
         <changelist>-SNAPSHOT</changelist>
-        <java.level>8</java.level>
-        <jenkins.version>2.263.1</jenkins.version>
+        <jenkins.version>2.387.3</jenkins.version>
```

Comment:
As recommended in [the documentation](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/).